src/adagrams.js

```diff
@@ -1,15 +1,92 @@
 export const drawLetters = () => {
   // Implement this method for wave 1
-};
+
+  const LETTER_POOL = {'A': 9,'B': 2,'C': 2,'D': 4,'E': 12,'F': 2,'G': 3,'H': 2,'I': 9,'J': 1,'K': 1,'L': 4,
```

Comment:
This can be declared outside of the function so that it does not have to be recreated each time the function is called.